### PR TITLE
[no ticket][risk=no] Fix e2e flaky tests

### PR DESCRIPTION
**Pull request by aweng98**
`owner-copy-to-workspace.spec.ts`:
CircleCI run [failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/16158/workflows/0118f1ec-601a-45ad-8932-c25502302f03/jobs/182862/tests).

`notebook-create-python.spec.ts`:
CircleCI run [failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/16161/workflows/663b03a4-9855-436e-9f0b-c4aa968466fb/jobs/182895/tests).

### CODE REVIEW

File: e2e/app/modal/copy-to-workspace-modal.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export default class CopyToWorkspaceModal extends Modal {
    */
   async copyToAnotherWorkspace(workspaceName: string, newName?: string): Promise<void> {
     await this.beginCopyToAnotherWorkspace(workspaceName, newName);
-    await this.clickButton(LinkText.Copy);
+    await this.clickButton(LinkText.Copy, { waitForClose: true });
```
Comment (aweng98) [inline e2e/app/modal/copy-to-workspace-modal.ts:62]: Wait for the Copy modal to close.

File: e2e/tests/nightly/owner-copy-to-workspace.spec.ts [hunk 2]
```diff
@@ -73,7 +73,10 @@ async function copyNotebookTest(sourceWorkspaceName: string, destCdrVersionName:
 
   // Copy to destination Workspace and give notebook a new name.
   const copiedNotebookName = makeRandomName('copy-of');
-  await analysisPage.copyNotebookToWorkspace(sourceNotebookName, destWorkspace, copiedNotebookName);
+  await analysisPage.copyNotebookToWorkspace(sourceNotebookName, destWorkspace, copiedNotebookName).catch(() => {
```
Comment (aweng98) [inline e2e/tests/nightly/owner-copy-to-workspace.spec.ts:76]: Retry again if `copyNotebookToWorkspace` fails and throws an exception.

File: e2e/libs/page-events-helper.ts [hunk 3]
```diff
@@ -76,7 +76,9 @@ export const shouldLogResponse = (request: Request): boolean => {
     '/data-set/',
     '/data-sets/',
     '/resources',
-    '/concept-sets/'
+    '/concept-sets/',
+    '/institutions',
+    '/admin/users/list'
```
Comment (aweng98) [inline e2e/libs/page-events-helper.ts:81]: Exclude two additional response bodies:  Large bodies, made test log very hard to read.

**Review by dolbeew — APPROVED** [review]
👍 Looks like both failures caused by failing request you mentioned on workbench-internal slack.

**Comment by aweng98** [comment]
Thank you.

**Pull request merged**